infrastructure/main.tf

```diff
@@ -23,79 +24,79 @@ data "azurerm_key_vault" "payment_key_vault" {
 
 data "azurerm_key_vault_secret" "gov_pay_keys_cmc" {
   name = "gov-pay-keys-cmc"
-  key_vault_id = "${data.azurerm_key_vault.payment_key_vault.id}"
+  key_vault_id = data.azurerm_key_vault.payment_key_vault.id
 }
 
 //created seperate kv for cmc claim store and copied claim store value
 resource "azurerm_key_vault_secret" "gov-pay-keys-cmc-claim-store" {
   name = "gov-pay-keys-cmc-claim-store"
-  value = "${data.azurerm_key_vault_secret.gov_pay_keys_cmc.value}"
-  key_vault_id = "${data.azurerm_key_vault.payment_key_vault.id}"
+  value = data.azurerm_key_vault_secret.gov_pay_keys_cmc.value
+  key_vault_id = data.azurerm_key_vault.payment_key_vault.id
 }
 
 
 module "payment-database" {
   source = "git@github.com:hmcts/cnp-module-postgres?ref=master"
   product = "${var.product}-postgres-db"
```

Comment:
You can modify product = "${var.product}-postgres-db" to **join("-", [var.product, "postgres-db"])**

This is to remove string interpolations.

Reply:
Done the interpolation changes